website/layouts/backend-types.erb

```diff
@@ -51,6 +51,9 @@
                   <li<%= sidebar_current("docs-backends-types-standard-manta") %>>
                     <a href="/docs/backends/types/manta.html">manta</a>
                   </li>
+                  <li<%= sidebar_current("docs-backends-types-standard-oss") %>>
```

Comment:
You don't have to include the sidebar_current thing anymore!

Reply:
Thanks!, I'll leave this in to keep it consistent, and we ca clean it all up at once later.